frontend-next/next.config.js

```diff
@@ -7,7 +7,7 @@ module.exports = withMDX({
 });
 
 module.exports = {
-    images: {
-      domains: ['raw.githubusercontent.com'],
-    },
-}  
+  images: {
+    domains: ['raw.githubusercontent.com'],
```

Comment:
we'll need to replace it with gitlab link or domain.


example: `https://bit.ly/3mEwSgP][https://gitlab.com/hackernoon/creative/-/raw/master/newsletters/logos/thematic/2020/November/flatfile-logo-black-small-vertical.png`

Reply:
@atherdon this was test for Image component from Next and I think it will not fit